core/bin/zksync_api/src/bin/providers/dev_liquidity_provider.rs

```diff
@@ -237,7 +237,9 @@ mod handlers_tests {
     #[actix_web::test]
     async fn get_coin_contract() {
         let expected_volume = BigDecimal::from_f32(2.34567).unwrap();
-        let token_address = "0xe978d5775bdCfB872c3Bf919Ce4169cDA6f8E271".to_string(); // an arbitrary, pre-generated address
+        let token_address = "0xe978d5775bdCfB872c3Bf919Ce4169cDA6f8E271"
+            .to_string()
+            .to_ascii_lowercase(); // an arbitrary, pre-generated address
```

Comment:
@jurajpiar I had to change this to lowercase, since in the implementation is done like that.

Reply:
👍